.githooks/post-checkout

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Can we add documentation in the file, to explain what this does.

Reply:
@deb1990 the files are pretty straightforward and they are fully documented in the CODING.md file, i think any additional comment here will be redundant